docs/meshstack.gcp.index.md

```diff
@@ -206,12 +206,17 @@ let GcpPlatformCoreConfiguration =
         The resulting string must not exceed a total length of 30 characters. Only alphanumeric + hyphen are allowed.
         We recommend that configuration include at least 3 characters of the random parameter to reduce the chance of
         naming collisions.
+
+      allowHierarchicalFolderAssignment:
```

Comment:
@tobiasweiss93 can you give a little bit more insight in the impact of the replication behaviour.

Reply:
I've extended the description.